test/cblas.h

```diff
@@ -0,0 +1,575 @@
+#ifndef CBLAS_H
+#define CBLAS_H
```

Comment:
Move this file to the external project clblas so as it is a third-party project and it is not part of sycl-blas